test/pdfs/issue14415.pdf.link

```diff
@@ -0,0 +1 @@
+https://github.com/mozilla/pdf.js/files/7803096/PDFJS_BUG.pdf
```

Comment:
This document looks small/simple enough, without any "sensitive" information, hence I think that you could just include directly in the repository instead.